src/iocore/dns/SplitDNSYAMLLoader.cc

```diff
@@ -0,0 +1,58 @@
+/** @file
+
+  Catch based unit tests for libinknet
+
+  @section license License
+
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+
+#include "iocore/dns/SplitDNSYAMLLoader.h"
+
+#include "tscpp/util/ts_errata.h"
+
+#include <swoc/Errata.h>
+#include <yaml-cpp/yaml.h>
+
+#include <sstream>
+#include <string>
+
+static std::string
+mark_to_error_message(YAML::Mark mark)
```

Comment:
No - use `Errata` and make a formatter for this. See [here](https://github.com/apache/trafficserver/blob/master/src/proxy/IPAllow.cc#L55). In fact, we might want to promote that to 'YamlCfg.h".

Reply:
Actually, see #10809